From 68859ed637de03c15ea4129f82920c324a3730f5 Mon Sep 17 00:00:00 2001
From: Chenxi Mao <chenxi.mao@suse.com>
Date: Thu, 22 Sep 2022 16:00:38 +0800
Subject: [PATCH] Revert "Makefile: link with -z noexecstack
 --no-warn-rwx-segments"
Patch-mainline: Never, revert stable patch which caused build-id lost
References: bsn#336
Modified-by-SEL: No


This reverts commit 8f4f2c9b98e40c699329680f9ca4f7c3a7f949ea.

Signed-off-by: Chenxi Mao <chenxi.mao@suse.com>
---
 Makefile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Makefile b/Makefile
index 4802db88e9d9..1730698124c7 100644
--- a/Makefile
+++ b/Makefile
@@ -983,9 +983,6 @@ KBUILD_CFLAGS   += $(KCFLAGS)
 KBUILD_LDFLAGS_MODULE += --build-id=sha1
 LDFLAGS_vmlinux += --build-id=sha1
 
-KBUILD_LDFLAGS	+= -z noexecstack
-KBUILD_LDFLAGS	+= $(call ld-option,--no-warn-rwx-segments)
-
 ifeq ($(CONFIG_STRIP_ASM_SYMS),y)
 LDFLAGS_vmlinux	+= $(call ld-option, -X,)
 endif
-- 
2.33.0

